### DISCUSSION TITLE
Recording with internal audio but not losing the sound output

### DISCUSSION THREAD

**Discussion by weehong-1** [ExistentialAudio/BlackHole#377] [Q&A]
Hello everyone,

Is it possible to record the screen with internal audio but not losing the functionality of listening to the audio. 

This is because sometimes I would need to record to meeting conversation and listening what they were discussing.

**Comment by devinroth** [comment]
Use a multi output.

Devin

**Reply by weehong-1** [reply]
Hi @devinroth,

Thank you for the prompt reply.

I managed to do it but I'm not sure if this is the proper way of doing it.
Would you mind to advise?

<img width="763" alt="Screenshot 2021-04-21 at 11 03 14 AM" src="https://user-images.githubusercontent.com/6753022/115490792-60a02e00-a291-11eb-865d-55a1f09d9d7d.png">
<img width="763" alt="Screenshot 2021-04-21 at 11 03 09 AM" src="https://user-images.githubusercontent.com/6753022/115490805-6564e200-a291-11eb-8d83-547759cbf5be.png">
